releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -6,6 +6,11 @@
 
 [Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 2.1.0 - September 06, 2021
+
+* Added Health Monitor support with Spring Actuator.
+* Added new API which allows users to configure the behavior when a Key Vault reference which cannot be resolved with `KeyVaultSecretProvider`.
```

Comment:
Anywhere people can find more details of the new API, or should we include more details here?

Reply:
Added so API details.